content/chainguard/administration/how-to-install-chainctl.md

```diff
@@ -71,6 +71,23 @@ sudo install -o $UID -g $GID -m 0755 chainctl /usr/local/bin/
 
 At this point, you'll be able to use the `chainctl` command.
 
+### Installing with `curl` in Windows Powershell
+
+Note that you can also use `curl` install `chainctl` on Windows systems. Running the following command in PowerShell will download the appropriate `.exe` file.
+
+```PowerShell
+curl -o chainctl.exe https://dl.enforce.dev/chainctl/latest/chainctl_windows_x86_64.exe
```

Comment:
This worked great, though took a long time. :+1:

Reply:
hmm, okay. It took a long time for me as well, I thought it might've just been my vm. Will add a note that it can take several minutes.